src/components/Background.tsx

```diff
@@ -17,46 +14,48 @@ import {
   OXYMORE_URL,
   PROJECTS_URL,
 } from "../constants/router-urls";
-import marbleBackground from "./assets/backgrounds/background.png";
-import eyeProjectBackground from "./assets/backgrounds/eye-bg.png";
+import styled, { css } from "styled-components";
+
 import Flex from "./Flex";
+import React from "react";
+import eyeProjectBackground from "./assets/backgrounds/eye-bg.png";
+import marbleBackground from "./assets/backgrounds/background.png";
+import { useLocation } from "react-router-dom";
 import { zIndexes } from "./theme";
 
-type Background = "black" | typeof marbleBackground;
+const black = "#020403";
```

Comment:
can you change the value of "black" in the theme and bring it in from there, similar to `zIndexes` above

Reply:
the value was the same, I was just repeating myself. good spot